lib/Yield/Pipes/CallGraphPipes.cpp

```diff
@@ -35,7 +35,10 @@ void ProcessCallGraph::run(pipeline::Context &Context,
   SortedVector<efa::FunctionMetadata> Metadata;
   for (const auto &LLVMFunction : FunctionTags::Isolated.functions(&Module))
     Metadata.insert(*extractFunctionMetadata(&LLVMFunction));
-  revng_assert(Metadata.size() == Model->Functions.size());
+
+  // If some functions are missing, do not output anything
+  if (Metadata.size() != Model->Functions.size())
+    return;
```

Comment:
Commit message is very vague. If you got these commits from other people, please ask them to provide an accurate commit message.